test/1_stdlib/VarArgs.swift

```diff
@@ -12,8 +12,8 @@ typealias CGFloat = Double
 #endif
 
 func my_printf(format: String, _ arguments: CVarArgType...) {
-  withVaList(arguments) {
-    vprintf(format, $0)
+  withVaList(arguments) { (vaList: CVaListPointer) in
+    vprintf(format, vaList)
```

Comment:
Exactly.  Unless there are subtle side effects to leaving it in, I really like the informative error messages that this change enables.